sapphire/tests/transformations/test_celestial.py

```diff
@@ -1,7 +1,9 @@
 import calendar
 import datetime
 import unittest
+import random as r
 from math import pi
+import matplotlib.pyplot as plt
```

Comment:
matplotlib is not a requirement for SAPPHiRE, it should not be used here.
It may be used in scripts in the `/sapphire/scripts` directory.